.github/ISSUE_TEMPLATE/4-release_checklist.md

```diff
@@ -10,47 +10,48 @@ assignees: ''
   - [ ] All tests pass in the ["Doctests" workflow](https://github.com/GenericMappingTools/pygmt/actions/workflows/ci_doctests.yaml)
-  - [ ] Deprecations and related tests are removed for this version by running `grep --include="*.py" -r 'remove_version="vX.Y.Z"' pygmt` from the base of the repository
-- [ ] Update warnings in `pygmt.show_versions()` as well as notes in [Common installation issues](https://www.pygmt.org/dev/install.html#not-working-transparency)
-      and [Testing your install]((https://www.pygmt.org/dev/install.html#testing-your-install) regarding GMT-Ghostscript incompatibility
+- [ ] Update warnings in `pygmt/_show_versions.py` as well as notes in
+      [Not working transparency](https://www.pygmt.org/dev/install.html#not-working-transparency)
+      regarding GMT-Ghostscript incompatibility
 - [ ] Reserve a DOI on [Zenodo](https://zenodo.org) by clicking on "New Version"
-- [ ] Review the ["PyGMT Team" page](https://www.pygmt.org/dev/team.html)
-- [ ] Finish up 'Changelog entry for v0.x.x' Pull Request:
-  - [ ] Add a new entry in `doc/_static/version_switch.js` for documentation switcher
-  - [ ] Update `CITATION.cff` and BibTeX at https://github.com/GenericMappingTools/pygmt#citing-pygmt
-    - [ ] Update authorship list
-    - [ ] Update DOI (and url for BibTeX)
-    - [ ] Update version
-    - [ ] Update date released
-  - [ ] Add the documentation link `doc/minversions.md`
-  - [ ] Add minimum required version information `doc/minversions.md`
-  - [ ] Copy draft changelog from Release Drafter and edit it to look nice ([see maintainers guide for details](https://www.pygmt.org/dev/maintenance.html#updating-the-changelog))
+- [ ] Finish up the "Changelog entry for v0.x.x" Pull Request (Use the previous changelog PR as a reference)
+- [ ] Run `make codespell` to check common misspellings. If there are any, either fix them or add them to `ignore-words-list` in `pyproject.toml`
+- [ ] Draft the announcement on https://hackmd.io/@pygmt
 
 **Release**:
+
 - [ ] At the [PyGMT release page on GitHub](https://github.com/GenericMappingTools/pygmt/releases):
   - [ ] Edit the draft release notes with the finalized changelog
   - [ ] Set the tag version and release title to vX.Y.Z
+  - [ ] Make sure "Set as the latests release" and "Create a discussion for this release" are selected
   - [ ] Make a release by clicking the 'Publish Release' button, this will automatically create a tag too
 - [ ] Manually upload the pygmt-vX.Y.Z.zip and baseline-images.zip files to https://zenodo.org/deposit, ensure that it is filed under the correct reserved DOI
 
 **After release**:
-- [ ] Update conda-forge [pygmt-feedstock](https://github.com/conda-forge/pygmt-feedstock) [Done automatically by conda-forge's bot, but remember to pin SPEC0 versions]
+
+- [ ] Update conda-forge [pygmt-feedstock](https://github.com/conda-forge/pygmt-feedstock)
```

Comment:
If conda-forge's bot takes much longer time to update the package, we can manually trigger the update by opening an issue with the title `@conda-forge-admin, please update version`. See https://github.com/conda-forge/pygmt-feedstock/issues/31 for an example.

Reply:
I noticed that the manually triggered update at https://github.com/conda-forge/pygmt-feedstock/pull/32/commits/1088e1ab94dede3655e3a253664b91ded65626b1 did not update the pin versions, whereas the auto-update at https://github.com/conda-forge/pygmt-feedstock/pull/30/commits/26a46092c82e1f3408d9beb935a2a8d39f214776 updated the SPEC 0 versions (via [grayskull](https://github.com/conda-forge/pygmt-feedstock/pull/27)).

Not sure if it's a bug on conda-forge's end, but maybe best if we just wait for the bot, and no need to document this `@conda-forge-admin, please update version` trigger here.